src/pages/examples.mdx

```diff
@@ -0,0 +1,22 @@
+import StarlightPage from "@astrojs/starlight/components/StarlightPage.astro";
+
+<StarlightPage
+  frontmatter={{
+    title: "Examples",
+    description: "Membrane program examples",
+  }}  
```

Comment:
We should also add a `headings` prop to autogenerate a ToC so that it's easy to view and jump to examples. The ToC wasn't generating when I tried, so I created a ticket to revisit it: [T-785](https://membrane.height.app/T-785) Add ToC